docs/pages/docs.md

```diff
@@ -5,20 +5,58 @@ layout: default
 
 # Documentation
 
-## Doc building
-
-| Name                                            | Short description                                               | 🚦  |
-| ----------------------------------------------- | --------------------------------------------------------------- | :-: |
-| [sphinx](https://www.sphinx-doc.org/en/master/) | Generates documentation from reStructuredText or Markdown files | 🟢  |
-| [gitbook](https://www.gitbook.com/)             | General documentation builder; integrates with GitHub           | 🟠  |
-| [mkdocs](https://www.mkdocs.org/)               | Generates documentation from Markdown files                     | 🟠  |
-| [pdoc](https://pdoc.dev/)                       | Auto-generates API documentation from docstrings                | 🟠  |
-
-<details>
-<summary> 🟢 explanation</summary>
-Sphinx is the de-facto standard that is widely used. It is well tested, reliable and very customisable.
+With Python, as with many other languages, it's very common to automatically
+create a web page with documentation built from [docstrings] in your package's
+API. For your package to be easily reusable we recommend you build documentation
```

Comment:
```suggestion
With Python, as with many other languages, it's very common to automatically
create a web page with documentation. This can include reference for the API taken
from your package's [docstrings], tutorials, examples, and more in depth explanation
about how the package works. For an in-depth discussion of the different components
of good documentation, see the [Diátaxis framework](https://diataxis.fr/).

For your package to be easily reusable we recommend you build documentation
```
Since API docs are not the only type of docs?

Reply:
Could not apply suggestion with the nice button. [c97ca4e](https://github.com/UCL-ARC/python-tooling/pull/327/commits/c97ca4edcde8990a6fb35cb104201046d87ad480)